### DISCUSSION TITLE
size of Sprite

### DISCUSSION THREAD

**Discussion by DrSeldom** [Bodmer/TFT_eSPI#2707] [Q&A - Sprites]
Hi

 background.createSprite(480,100); - its WORK!
 background.createSprite(480,120); - dont work (

Arduino IDE 2
TFT_eSPI ver 2.5.0
ESP32
#define RPI_ILI9486_DRIVER // 20MHz maximum SPI                        //480x320
SPI

when it works everything is fine, if it doesn't work then the screen is white

**Comment by Bodmer** [comment]
The maximum sprite size is determined by the available RAM. See here in ReadMe for more info:

https://github.com/Bodmer/TFT_eSPI#sprites

**Comment by DrSeldom** [comment]
Thanks!
